esp_wrover_kit/esp_wrover_kit.c

```diff
@@ -98,6 +98,51 @@ static void lvgl_port_flush_callback(lv_disp_drv_t *drv, const lv_area_t *area,
     esp_lcd_panel_draw_bitmap(panel_handle, offsetx1, offsety1, offsetx2 + 1, offsety2 + 1, color_map);
 }
 
+static void lvgl_port_update_callback(lv_disp_drv_t *drv)
+{
+    esp_lcd_panel_handle_t panel_handle = (esp_lcd_panel_handle_t) drv->user_data;
+
+    switch (drv->rotated) {
+    case LV_DISP_ROT_NONE:
+        // Rotate LCD display
+        esp_lcd_panel_swap_xy(panel_handle, false);
+#ifdef CONFIG_BSP_LCD_ILI9341
+        esp_lcd_panel_mirror(panel_handle, true, false);
+#else
+        esp_lcd_panel_mirror(panel_handle, false, false);
+#endif
+        esp_lcd_panel_mirror(panel_handle, true, true);
+        break;
+    case LV_DISP_ROT_90:
+        // Rotate LCD display
+        esp_lcd_panel_swap_xy(panel_handle, true);
+#ifdef CONFIG_BSP_LCD_ILI9341
+        esp_lcd_panel_mirror(panel_handle, false, false);
+#else
+        esp_lcd_panel_mirror(panel_handle, true, false);
+#endif
+        break;
+    case LV_DISP_ROT_180:
+        // Rotate LCD display
+        esp_lcd_panel_swap_xy(panel_handle, false);
+#ifdef CONFIG_BSP_LCD_ILI9341
+        esp_lcd_panel_mirror(panel_handle, false, true);
+#else
+        esp_lcd_panel_mirror(panel_handle, true, true);
+#endif
+        break;
+    case LV_DISP_ROT_270:
+        // Rotate LCD display
+        esp_lcd_panel_swap_xy(panel_handle, true);
+#ifdef CONFIG_BSP_LCD_ILI9341
+        esp_lcd_panel_mirror(panel_handle, true, true);
+#else
+        esp_lcd_panel_mirror(panel_handle, false, true);
+#endif
+        break;
```

Comment:
Same as in esp-box. 90/270 rotation swapped? 
https://docs.lvgl.io/latest/en/html/porting/display.html#rotation

Reply:
Fixed on all boards.